documentation/puppet-api/v3/environment_modules.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "Environment Modules",
+  "description": "Information about the modules in a Puppet code environment",
+  "type":        "object",
+  "properties": {
+    "modules": {
+      "description": "The array of modules which exist in an environment.",
+      "type": "array",
+      "properties": {
+        "name": {
+          "description": "The name of the puppet module",
+          "type": "string"
+        },
+        "version": {
+          "description": "The version of the puppet module",
+          "type": "string"
+        }
+      }
+    },
+    "name": {
+      "description": "Name of the environment",
+      "type": "string"
+    }
+  }
+}
```

Comment:
@camlow325 that seems pretty solid to me. I can update the PR with this new schema.